src/plugins/console/public/styles/_app.scss

```diff
@@ -145,17 +143,6 @@
   margin-inline: 0;
 }
 
-// SASSTODO: This component seems to not be used anymore?
```

Comment:
I think this file has some more code that can be deleted (e.g. the whole `.ace_badge` class).

Reply:
Ohh nice catch, I missed that one!